migrations/versions/840_add_application_close_date_for_framework.py

```diff
@@ -0,0 +1,37 @@
+"""Add `application_close_date` for Framework
+Revision ID: 840
+Revises: 830
+Create Date: 2017-02-06 11:09:26.852142
+"""
+from alembic import op
+from datetime import datetime
+import sqlalchemy as sa
+
+from dmutils.formats import DATETIME_FORMAT, EUROPE_LONDON
+
+# revision identifiers, used by Alembic.
+revision = '840'
+down_revision = '830'
+
+
+
+def upgrade():
+    op.add_column('frameworks', sa.Column('application_close_date', sa.DateTime(), nullable=True))
+    op.add_column('frameworks', sa.Column('allow_declaration_reuse', sa.Boolean(), nullable=False, server_default='false'))
+
+    for framework_slug, a_c_date, declaration_reuse in (
+        ('digital-outcomes-and-specialists', datetime(2016, 1, 1, 15, tzinfo=EUROPE_LONDON), False),
+        ('digital-outcomes-and-specialists-2', datetime(2017, 1, 16, 17, tzinfo=EUROPE_LONDON), True),
+        ('g-cloud-8', datetime(2016, 6, 1, 17, tzinfo=EUROPE_LONDON), True),
+
+    ):
+        query_string = """UPDATE frameworks set application_close_date = '{a_c_date}'::DATE where slug = '{framework_slug}'"""
+        op.execute(query_string.format(framework_slug=framework_slug, a_c_date=a_c_date.strftime(DATETIME_FORMAT)))
```

Comment:
I'm not sure why you're searching by `name` and not `slug`, secondly, it shouldn't be problematic as it's coming from an id field that should already be constrained to an integer. But still not something to encourage. ORMs exist. Database adaptors can do proper safe parametrization. Don't ignore them.